frameworks/g-cloud-9/questions/services/multiqAPIHosting.yml

```diff
@@ -1,18 +1,17 @@
+id: multiqAPI
```

Comment:
Actually, if we're using these for slug generation is there danger of URL clash at all?